### PR TITLE
Fix global testMiddlewarePath writing out to /tmp

### PR DESCRIPTION
**Pull request by titpetric**
Fixes tests code to not write out a testMiddlewarePath to /tmp on gateway start.

Intended for: master, LTS, 5.3

### CODE REVIEW

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5380/merge
Commit:  
Triggered by: pull_request (@titpetric)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5730233611)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5380/merge
Commit: 31ac1bb5017926ac86bd10ff38352327c09cf6c9 
Triggered by: pull_request (@titpetric)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5730728004)

File: gateway/coprocess_bundle_test.go [hunk 1]
```diff
@@ -60,6 +56,7 @@ func TestBundleLoader(t *testing.T) {
 	})
 
 	t.Run("Existing bundle with auth check", func(t *testing.T) {
+		testBundlesPath := filepath.Join(ts.Gw.GetConfig().MiddlewarePath, "bundles")
```
[the remark this answers is not in the document]
Comment (buger) [reply gateway/coprocess_bundle_test.go:59]: 👎🏻

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5380/merge
Commit: 80c6699d28d4f677e05113d18dfc465a53b501c2 
Triggered by: pull_request (@buger)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5735568686)

**Review by andrei-tyk — APPROVED** [review]

**Pull request closed without merging**